Automattic/wp-calypso
# Congrats: Refactored domain bulk transfer congrats page

Related to https://github.com/Automattic/dotcom-forge/issues/5121

## Proposed Changes

This PR refactors the Bulk Transfer Congrats Page to adapt the new approach for unifying the congratulation pages using the `<ThankYouV2>` component. See pb5gDS-38y-p2#comment-4475 for details.

## Testing Instructions



* If you don't have a domain previously purchased, purchase one.
* If you have a domain handy, get the respective receipt ID using the following steps:
    * Go to Upgrades > Purchases > Billing History
    * Look for the domain purchase and click on "View receipt" and you should be able to find the receipt ID
* Point `public-api.wordpress.com` to your sandbox IP.
* Apply the following diff 332e2-pb/#diff to your sandbox to force the purchase to be returned as a domain transfer:
* Visit `/checkout/thank-you/:site/:receipt_id` and assert that you see the Domain Transfer Page.
* Compare the same page in production and assert that you see the same page content, with the exception of the following:
    * Spacing is a bit different, since we migrated to the unified template.
    * ~The confetti animation is removed.~ [**UPDATE**: We are keeping the animation after all p9Jlb4-7Hx-p2#comment-10513] 
* Assert that all buttons and links click as expected.

<img width="1110" alt="Screen Shot 2024-01-11 at 4 34 05 PM" src="https://github.com/Automattic/wp-calypso/assets/104910361/c8165afd-3e48-46a8-b7a1-535944a44769">

* Apply the following diff 332e3-pb/#diff to your sandbox to force a purchase of `$0`:
* Visit `/checkout/thank-you/:site/:receipt_id` and assert that you see the Domain Transfer Page and that it functions the same way as the production page with the same remarks noted above regarding spacing / confetti animation.

<img width="1100" alt="Screen Shot 2024-01-11 at 4 42 08 PM" src="https://github.com/Automattic/wp-calypso/assets/104910361/6c5934ca-ee16-4e80-99e3-11baf00bd8bc">

## Pre-merge Checklist
